generator/template.go

```diff
@@ -223,7 +223,7 @@ export function replacer(key: any, value: any): any {
   return value;
 }
 
-export function fetchReq<I, O>(path: string, init?: InitReq): Promise<O> {
+export function fetchRequest<R>(path: string, init?: any): Promise<R> {
```

Comment:
The input type wasn't being used and actually led to inaccurate type definitions because of how `init` is constructed. For example, here is a PATCH request:

```ts
static HTTPPatch(req: HttpPatchRequest, initReq?: fm.InitReq): Promise<HttpPatchResponse> {
    return fm.fetchRequest<HttpPatchResponse>(`/patch`, {...initReq, method: "PATCH", body: JSON.stringify(req, fm.replacer)})
  }
```

Reply:
This seems like a good change, but since it's exported it will require a "breaking changes" type changelog note when we do a release, I think?

Reply:
I _think_ this was meant to be considered internal, so shouldn't be called directly with application code, but that is a good call out.